gnetlist/src/s_traverse.c

```diff
@@ -357,7 +357,7 @@ s_traverse_net (NET *nets, int starting, OBJECT *object, char *hierarchy_tag, in
 
   if (object->type == OBJ_PIN) {
 
-    verbose_print (starting ? "p" : "P");
+    starting ? verbose_print ("p") : verbose_print ("P");
```

Comment:
What was the warning that this change fixes?  The original code looks okay to me, so I must be missing something important!